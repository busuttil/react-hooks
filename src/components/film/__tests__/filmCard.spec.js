import React from 'react';
import { shallow } from 'enzyme';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import FilmCard from '../filmCard';

describe('FilmCard', () => {
  let props = {
    film: {
      id: 1,
      title: 'title',
      vote_average: 10,
    },
  };

  const getWrapper = () => shallow(<FilmCard {...props} />);

  it('render card', () => {
    expect(getWrapper());
  });
});
