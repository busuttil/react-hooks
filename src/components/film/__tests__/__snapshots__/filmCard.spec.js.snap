// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCard render card 1`] = `ShallowWrapper {}`;
